File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.java
class Solution {
    public int solution(String[] strArr) {
        int answer = 0;
        
        int[] arr = new int[31];
        
        for(int i = 0; i < strArr.length; i++){
            int len = strArr[i].length();
            answer = answer < ++arr[len] ? arr[len] : answer;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.py
def solution(strArr):
    answer = 0
    
    arrlen = [0] * 31
    
    for str in strArr:
        arrlen[len(str)] += 1
        
        if answer < arrlen[len(str)]:
            answer = arrlen[len(str)]
    
    return answer